src/main/java/seedu/address/model/lesson/MakeUpLesson.java

```diff
@@ -89,5 +101,24 @@ public boolean isCancelled() {
     public boolean hasLessonOnDate(Date date) {
         return getStartDate().equals(date) && !isCancelled();
     }
+
+    @Override
+    public String toString() {
+        final StringBuilder builder = new StringBuilder();
+
+        builder.append("(Makeup)")
```

Comment:
Can use getTypeOfLesson() here so no magic strings but since the others already have magic strings, not that important